### DISCUSSION TITLE
USB to COM conversion in an ubuntu 16.04 64bit environment.

### DISCUSSION THREAD

**Discussion by jkfuerte** [Stellarium/stellarium#136] [Q&A] [answered]
I just successfully configure a windows 10 pc to interface Stellarium to a NetStar telescope. After installing the ASCOM plataform and related driver for that telescope, the major problem was the driver to convert the USB interface to a COM port. Under Windows 10, I opened the devices management window, I found the USB to COM element of the tree and I clicked on the "search a driver for it" button.
How can I solve this in a linux ubuntu sysytem?

**Comment by alex-w** [comment]
Without knowledge about your hardware to give an answer is impossible. In generic case maybe your device is available already in /dev/ttyUSB* or installing INDI server may solve the problem.

**Comment by gzotti** [comment, marked as answer]
Normally udev will discover the USB device and create a device node like /dev/ttyUSB0 (as Alexander has written). Open a terminal, plugin your USB-to-serial device and use 'dmesg' to see diagnostics. Some such devices have been disabled from recognition, though. This is not a Stellarium-related issue, but general Linux/USB-to-serial. Google should give you more hits on that.
